src/components/ColorPicker/components/ColorPickerContent/ColorPickerContentComponent.jsx

```diff
@@ -159,7 +162,8 @@ ColorPickerContentComponent.defaultProps = {
    * Used to force the component render the colorList prop as is. Usually, this flag should not be used. It's intended only for edge cases.
    * Usually, only "monday colors" will be rendered (unless blacklist mode is used). This flag will override this behavior.
    */
-  forceUseRawColorList: false
+  forceUseRawColorList: false,
+  showColorNameTooltip: false
```

Comment:
please add description for the new prop.  
Also, if it's incompatible with `forceUseRawColorList`, please mention it.  
(by the way - why is it incompatible? 🤔 )  

And lastly, consider explaining what happens if `tooltipContentByColor` is supplied while `showColorNameTooltip` is truthy (`tooltipContentByColor` will take precendence)

Reply:
@laviomri it's incompatible with forceUseRawColorList cause it relies on the color structure (that it has name)